### PR TITLE
General questions

### PR DESCRIPTION
**Pull request by mouchourider**
Creation of general question form + send answers to the firebase DB (Not authentified user)

### CODE REVIEW

File: src/components/startQs.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const StartQs = (props) => {
+    return (
+      <div className="startQs">
+      <h1 className="container">הקדמה</h1>
+      <div className="container">
+        <h2>רגע לפני שמתחילים</h2>
+        <div>יש לנו שאלון קצר שיעוזר לנו להתאים עבורכם את העזרה</div>
+      </div>
+      <div className="buttom-container">
+        <button className="button-with-effect" onClick={event => window.location.href='/generalQS/general'}> שאלון התאמה</button><br /><br />
```
Comment (leechechik) [inline src/components/startQs.jsx:14]: please change from window.location.href --> to history.puth, it's a bad practice to use the window.location.href and can cause us problems.

File: src/components/questionnarie/generalQS.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+              open={modalResultsState}
+              closeOnDocumentClick
+              onClose={closeModalResults}
+        >
+          <div className="popup-container" style={{display: 'flex',  justifyContent:'center', alignItems:'center', height: '100vh'}}>
+            <h1 className="big-text">תודה על תשובותיך!</h1>
+            <div className="small-text">אנחנו מתאימים עבורך את השאלון</div>
+            <div><Ring color="#be97e8" size={200} /></div>
+
+            <button className="button-general" onClick={event => window.location.href='/SelectRoom'}>בחירת חדרים</button>
```
Comment (leechechik) [inline src/components/questionnarie/generalQS.jsx:103]: Same here, please change to history.push (reference - https://reacttraining.com/react-router/web/api/Hooks/usehistory)

File: src/components/questionnarie/generalQS.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+  const onSubmit = (answer) => {
+    
+    questions[currQuestion].answer = answer;
+    const nextQuestion = currQuestion + 1;
+
+    if (nextQuestion === questions.length){
+      const answers = questions.map(x => ({ questionId:x.id, answer: x.answer}));
+      // save answers to DB
+      saveAnswersGeneral('test_user_id', answers);
```
Comment (leechechik) [inline src/components/questionnarie/generalQS.jsx:48]: please change this to the user authenticated, and remove this 'test_user_id'

Comment (mouchourider) [reply src/components/questionnarie/generalQS.jsx]: Solved!

File: src/components/questionnarie/generalQS.jsx (hunk 2 above)
Comment (mouchourider) [reply src/components/questionnarie/generalQS.jsx]: Solved!

File: src/components/startQs.jsx (hunk 1 above)
Comment (mouchourider) [reply src/components/startQs.jsx]: Solved!

**Comment by mouchourider** [comment]
> 
> 
> @mouchourider Something looks wrong in the branch, you have here changes which don't belong to you and not updated.
> I suggest moving your comments to a new branch.
> In addition, please see my comments here :)
I  know but I was asked during a previous meeting to add Shoval's room selection screen to my branch because the "General question" screen should leads to this screen in the flow diagram.

**Pull request closed without merging**